Source/External/ExCSS/StylesheetReader.cs

```diff
@@ -23,7 +23,7 @@ internal class StylesheetReader
 
         internal StylesheetReader(string styleText) : this()
         {
-            _reader = new StringReader(styleText);
+            _reader = new StringReader(styleText??string.Empty);
```

Comment:
@BenLampson 

Thanks for this PR !!
However, please add it to the following source file instead of this externally provided source file.
https://github.com/svg-net/SVG/blob/f631457d4127d4779c479dd980a1bbb400f9b9b7/Source/SvgDocument.cs#L490

```cs
var sheet = cssParser.Parse(cssTotal ?? string.Empty);
```

Reply:
Thanks for this comment and your tips, StylesheetReader is an internal constructor, so we need not check the null case.